server/user.go

```diff
@@ -33,7 +33,8 @@ type Connection struct {
 	Oauth1AccessSecret string        `json:",omitempty"`
 	OAuth2Token        *oauth2.Token `json:",omitempty"`
 	Settings           *ConnectionSettings
-	DefaultProjectKey  string `json:"default_project_key,omitempty"`
+	DefaultProjectKey  string   `json:"default_project_key,omitempty"`
+	MattermostUserID   types.ID `json:"mattermost_user_id"`
```

Comment:
We will want to make sure this change is backwards compatible. This value not be present for existing users

Reply:
Yes, for existing Cloud and Server Instance, this value will be nil. Currently we are only using this when a user creates a new Cloud Oauth instance. It is not used for any other instance type and can remain to be nil.